libraries/AP_HAL/RCOutput.h

```diff
@@ -222,6 +222,9 @@ class AP_HAL::RCOutput {
     // static to allow use in the ChibiOS thread stuff
     static bool is_dshot_protocol(const enum output_mode mode);
 
+    static bool is_led_protocol(const enum output_mode mode) {
+      return mode == MODE_NEOPIXEL || mode == MODE_PROFILED;
```

Comment:
This should be a switch statement so if we get a new protocol the user has to add it in here.